app/views/statements/list.html.haml

```diff
@@ -4,11 +4,11 @@
   = t(".head.h1")
 
 %h2= @account
-%h3= t(".head.h3", facility: @facility.to_s)
+%h3= text("statements.head.h3", facility: @facility.to_s)
```

Comment:
Is this correct?
```suggestion
%h3= text("statements.list.head.h3", facility: @facility.to_s)
```

Reply:
I'm not sure if this file is actually used - @jossim were you able to test this one in the browser?